add-on/src/lib/ipfs-companion.js

```diff
@@ -453,12 +466,47 @@ export default async function init (inQuickImport = false) {
   // API STATUS UPDATES
   // -------------------------------------------------------------------
   // API is polled for peer count every ipfsApiPollMs
+  // In Manifest V3 (Chromium), service workers go dormant after ~30 seconds of inactivity,
+  // which stops setInterval. To ensure continuous monitoring, we use a hybrid approach:
+  // - For polling < 30s: Use BOTH setInterval (for precise timing) AND alarms (as backup)
+  // - For polling >= 30s: Use ONLY alarms (avoids duplicate timers, more efficient)
+  // Note: Chrome alarms API has a minimum interval of 30 seconds
 
   async function setApiStatusUpdateInterval (ipfsApiPollMs) {
+    // Clear existing interval if present
     if (apiStatusUpdateInterval) {
       clearInterval(apiStatusUpdateInterval)
+      apiStatusUpdateInterval = null
+    }
+
+    // Clear existing alarm if present
+    if (browser.alarms) {
+      await browser.alarms.clear(kuboRpcStatusAlarmName)
+    }
+
+    // Use alarms API if available (for Manifest V3 compatibility)
+    if (browser.alarms) {
+      // Always create an alarm as backup (minimum 30 seconds)
+      const periodInSeconds = Math.max(30, Math.floor(ipfsApiPollMs / 1000))
+      await browser.alarms.create(kuboRpcStatusAlarmName, {
+        periodInMinutes: periodInSeconds / 60
+      })
+      log(`Created alarm for API status updates every ${periodInSeconds} seconds`)
```

Comment:
*readability*

since the API takes periodInMinutes anyway, why not just create `periodInMinutes` ?

```suggestion
      const periodInMinutes = Math.max(30, Math.floor(ipfsApiPollMs / 1000)) / 60
      await browser.alarms.create(kuboRpcStatusAlarmName, {
        periodInMinutes
      })
      log(`Created alarm for API status updates every ${periodInMinutes} minutes`)
```

I had to look and see if there was some reason for periodInSeconds other than the log item.